collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -208,6 +209,19 @@ public String getVersionedAppName() {
         return getString(R.string.app_name) + versionName;
     }
 
+    /**
+     * Get a User-Agent string that provides the platform details followed by the application ID
+     * and application version name: {@code Dalvik/<version> (platform info) org.odk.collect.android/v<version>}.
+     *
+     * This deviates from the recommended format as described in https://github.com/opendatakit/collect/issues/3253.
```

Comment:
Not a big deal, but I like putting the app ID first, at least in the long term.  Did you keep `Dalvik` first because you'd prefer to keep it first, or just to minimize changes for now?

Reply:
Yes, I do as well. I thought it was nice to have a single place where we generate a User-Agent string. And since changing it for all HTTP requests is not really a fix, I'd rather do it after v1.23 is out. I don't think changing it for osmdroid then will have any repercussions.